tools/build_travis.py

```diff
@@ -326,6 +334,20 @@
                        }
              }
         )
+    },
+    {
+        "RENESAS": (
+            {"target": "RZ_A1H",
+             "toolchains": "GCC_ARM",
+             "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_16"],
+                       }
+             },
+            {"target": "VK_RZ_A1H",
+             "toolchains": "GCC_ARM",
+             "tests": {""     : ["MBED_2", "MBED_10", "MBED_11", "MBED_16"],
+                       }
```

Comment:
NIT: extra space

Reply:
Once travis finishes I can fix these, I used formatting that was in the file (many of the lines have this).